### PR TITLE
PE-1221: Missing Spanish translations

### PR DESCRIPTION
**Pull request by matibat**
**This PR...**
- Adds the missing keys at app_es.arb
  - So the Back and Next buttons on the onboarding page are translated
  - The form validation messages are translated 
  - The manifest creation form is fully translated
- Fixes a Spanish typo: "Aplicación" at the onboarding page
- Declares insufficientARWarning and displayedPaginatedData keys in both ARB files
  - So the "Insufficient Funds" warning is translated
  - The paginated data count is translated: {from} - {to} of {total}
- Temporarily adds the keys aggreeToTerms_main, aggreeToTerms_link, welcomeTo_main, and welcomeTo_description directly in the code as those has a direct order relationship (1 to 1) between en and es. That will be dynamically handled in a [following-up work PE-1125](https://ardrive.atlassian.net/browse/PE-1125) when required for other languages.

### CODE REVIEW

File: lib/l10n/app_es.arb [hunk 1]
```diff
@@ -210,6 +214,10 @@
   "@next": {
     "description": "Action of going to the next page"
   },
+  "nextEmphasized": "NEXT",
```
Comment (matibat) [inline lib/l10n/app_es.arb:217]:
```suggestion
  "nextEmphasized": "SIGUIENTE",
```

File: lib/l10n/app_es.arb [hunk 2]
```diff
@@ -765,6 +773,10 @@
   "@back": {
     "description": "Go back"
   },
+  "backEmphasized": "BACK",
```
Comment (matibat) [inline lib/l10n/app_es.arb:776]: This is for text on the onboarding page, the BACK button. I suggest:
```suggestion
  "backEmphasized": "VOLVER",
```
```suggestion
  "backEmphasized": "ATRÁS",
```

File: lib/l10n/app_es.arb [hunk 3]
```diff
@@ -811,6 +823,10 @@
   "@upload": {
     "description": "The action of uploading certain item"
   },
+  "uploadEmphasized": "UPLOAD",
```
Comment (matibat) [inline lib/l10n/app_es.arb:826]:
```suggestion
  "uploadEmphasized": "CARGAR",
```

**Review by matibat — COMMENTED** [review]
A couple more

File: lib/l10n/app_es.arb (hunk 2 above)
Comment (matibat) [reply lib/l10n/app_es.arb:776]:
```suggestion
  "backEmphasized": "ANTERIOR",
```

**Review by karlprieb — APPROVED** [review]

**Review by fedellen — APPROVED** [review]
The Spanish UI view is looking nice!

File: lib/l10n/app_es.arb [hunk 4]
```diff
@@ -1030,5 +1046,156 @@
...
+  },
+  "walletChangedDuringManifestCreation": "El monedero que estabas usando cambió mientras se creaba el manifesto...",
+  "@walletChangedDuringManifestCreation": {
+    "description": "The wallet has been changed while the creation of the manifest was in process"
+  },
+  "manifestTransactionUnexpectedlyFailed": "La transacción del manifesto ha fallado inesperadamente mientras se cargaba a la red de Arweave...",
+  "@manifestTransactionUnexpectedlyFailed": {
+    "description": "The manifest transaction did not succeed being uploaded"
+  },
+  "insufficientBalanceForManifest": "El balance de {walletBalance} AR es insuficiente para esta transacción de manifesto que cuesta: {totalCost} AR...",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1074]: manifiesto

File: lib/l10n/app_es.arb [hunk 5]
```diff
@@ -1030,5 +1046,156 @@
...
+      }
+    }
+  },
+  "conflictingNameFound": "Se encontró un conflicto de nombres",
+  "@conflictingNameFound": {
+    "description": "There is a non-manifest entity with the same name"
+  },
+  "conflictingNameFoundChooseNewName": "Una entidad con ese nombre ya existe en este lugar. Por favor, elige un nombre diferente.",
+  "@conflictingNameFoundChooseNewName": {
+    "description": "The user must type a non-conflicting name"
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1094]: Update description to reflect text

File: lib/l10n/app_es.arb [hunk 6]
```diff
@@ -1030,5 +1046,156 @@
...
+  },
+  "conflictingManifestFound": "El manifesto ya existe",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1096]: manifiesto

File: lib/l10n/app_es.arb [hunk 7]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@conflictingManifestFound": {
+    "description": "There is an on-chain entity with the same name"
+  },
+  "conflictingManifestFoundChooseNewName": "Ya hay un manifesto con ese nombre aquí. ¿Quieres continuar y cargar este manifesto como una nueva revisión?",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1100]: manifiesto

File: lib/l10n/app_es.arb [hunk 8]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@conflictingManifestFoundChooseNewName": {
+    "description": "A manifest with the same name already exists. Ask the user for confirmation about uploading it as a new revision"
+  },
+  "addnewManifestEmphasized": "AÑADIR NUEVO MANIFESTO",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1104]: MANIFIESTO

File: lib/l10n/app_es.arb [hunk 9]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@addnewManifestEmphasized": {
+    "description": "There is a non-manifest entity with the same name"
+  },
+  "aManifestIsASpecialKindOfFile": "Un manifesto es un tipo especial de archivos que relaciona cualquier cantidad de transacciones Arweave con direcciones amigable.",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1108]: "Un manifiesto es un tipo especial de archivo que relaciona transacciones Arweave con rutas perzonalizadas."

File: lib/l10n/app_es.arb [hunk 10]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@aManifestIsASpecialKindOfFile": {
+    "description": "Simple explanation of what a manifest is"
+  },
+  "learnMore": "Conocer más",
+  "@learnMore": {
+    "description": "Link to a document explaining manifests in detail"
+  },
+  "createManifestEmphasized": "CREAR MANIFESTO",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1116]: MANIFIESTO

File: lib/l10n/app_es.arb [hunk 11]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@createManifestEmphasized": {
+    "description": "To create a manifest"
+  },
+  "filesWillBePermanentlyPublicWarning": "Todos los archivos cargados aquí serán permanentemente públicos para cualquiera en el internet. Asegúrate de estar haciendo la información pública a propósito.",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1120]: "Todos los archivos cargados aquí serán públicos por siempre para cualquiera en el internet. Asegúrate de que quieres que esta informacion sea publica."

File: lib/l10n/app_es.arb [hunk 12]
```diff
@@ -1030,5 +1046,156 @@
...
+  },
+  "validationAttachDriveCouldNotBeFound": "No se encontró la unidad especificada.",
+  "@validationAttachDriveCouldNotBeFound": {
+    "description": "Input validation error: The given drive ID does not exist"
+  },
+  "validationInvalidKey": "Clave de la unidad inválida.",
+  "@validationInvalidKey": {
+    "description": "Input validation error: The given drive key is wrong"
+  },
+  "validationAttachUserLoggedOut": "Debes iniciar sesión para vincular discos privados.",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1160]: ... vincular unidades privadas."

File: lib/l10n/app_es.arb [hunk 13]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@validationAttachUserLoggedOut": {
+    "description": "Input validation error: the user cannot attach a drive when not logged in"
+  },
+  "validationEntityNameAlreadyPresent": "Una carpeta o archivo con el mismo nombre ya existe aquí.",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1164]: "Ya existe una carpeta o archivo con el mismo nombre."

File: lib/l10n/app_es.arb [hunk 14]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@validationEntityNameAlreadyPresent": {
+    "description": "Input validation error: The given name is already in use by some exising entity"
+  },
+  "validationDriveNameAlreadyPresent": "Un disco con el mismo nombre ya existe aquí.",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1168]: "Una unidad con el mismo nombre ya existe."

File: lib/l10n/app_es.arb [hunk 15]
```diff
@@ -1030,5 +1046,156 @@
...
+  "@validationDriveNameAlreadyPresent": {
+    "description": "Input validation error: The given name is already in use by some exising drive"
+  },
+  "validationNameUnchanged": "Este nombre es el idéntico al nombre original",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:1172]: "Este nombre es idéntico al nombre original"

**Review by agsuy — CHANGES_REQUESTED** [review]
Left some comments

**Review by jdaev — APPROVED** [review]

**Review by agsuy — APPROVED** [review]
LGTM!

**Pull request merged**